docs/Nodejs.md

```diff
@@ -0,0 +1,47 @@
+# Node.js
+
+## About
+
+
+Node.js is an asynchronous event driven framework that provides an event-driven architecture and a non-blocking I/O API designed to optimize an application's throughput and scalability for real-time applications. It uses Google V8 JavaScript engine to execute code, and a large percentage of the basic modules are written in JavaScript.  
+
+Node's non-blocking model makes for a perfect framework to use products like [Phidgets](http://www.phidgets.com/) where one may want many devices communicating and reporting changes without having to worry about other devices or processes blocking the flow thus allowing for real-time, dynamic applications.
+
+## Installation
+It is advisory to have the latest version of Node.js for best performance of this API.
+
+Node.Js is available on all platforms including both 64 and 32 bit architecture:
+
+
+1. [Linux](https://nodejs.org/en/)
+
+2. [Mac OS](https://nodejs.org/en/)
+
+3. [Widnows](https://nodejs.org/en/)
+
+### Linux
+#### Source Code
+* Download and unpack the Node.js Source code [available here](https://nodejs.org/en/download/)
+    * From the main unpacked libraries directory, run:
+
+        `./configure`
+
+        `make`
+
+        `sudo make install`
+
+#### Binaries
+* Download the [binary file](https://nodejs.org/en/download/) for your system and run the Installer.
+
+### Mac OS
+* Download the [Macintosh Installer](https://nodejs.org/en/download/) for your system and run the Installer.
```

Comment:
Mac is also a unix based system so can install the same way as linux I believe. Have to give it a shot.